modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowServiceImpl.java

```diff
@@ -1183,6 +1136,16 @@ protected void assertPermission(WorkflowInstance workflow, String action, String
     }
   }
 
+  protected boolean assertMediaPackagePermission(String mediaPackageId, String action) {
+    Opt<MediaPackage> assetMediaPackage = assetManager.getMediaPackage(mediaPackageId);
+    if (assetMediaPackage.isSome()) {
+      var currentUser = securityService.getUser();
+      var mediaPackage = assetMediaPackage.get();
+      return currentUser.hasRole(GLOBAL_ADMIN_ROLE) || authorizationService.hasPermission(mediaPackage, action);
```

Comment:
Should this also be checking the org admin roles?  From the looks of things `authorizationService` doesn't...